lib/consts.ts

```diff
@@ -1,20 +1,22 @@
 import { homedir } from "os";
 import path from "path";
 
-export const API_HOST =
-  process.env.DITTO_API_HOST || "https://api.dittowords.com";
-export const CONFIG_FILE =
-  process.env.DITTO_CONFIG_FILE || path.join(homedir(), ".config", "ditto");
-export const PROJECT_CONFIG_FILE = path.normalize(
-  path.join("ditto", "config.yml")
-);
-export const TEXT_DIR = process.env.DITTO_TEXT_DIR || "ditto";
-export const TEXT_FILE = path.normalize(path.join(TEXT_DIR, "text.json"));
-
-export default {
-  API_HOST,
-  CONFIG_FILE,
-  PROJECT_CONFIG_FILE,
-  TEXT_DIR,
-  TEXT_FILE,
-};
+export default new (class {
```

Comment:
In all places but one we already imported the default export of this file (and I refactored that one place), so this doesn't change anything functionally, but we can utilize the ES6 setter to allow us to dynamically set these values in tests.